build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+import org.ajoberstar.grgit.Grgit
```

Comment:
Why did you convert the build script to Kotlin DSL? That should really be in its own PR.

Reply:
See #295 and the description of this PR.
I cleaned the build scripts because the existing scripts didn't work. E.g. i was unable to build the universal jar file.